plugins/exporter.koplugin/clip.lua

```diff
@@ -242,6 +242,7 @@ function MyClipping:parseHighlight(highlights, bookmarks, book)
                                "(.*)",
                                "%d%d%d%d%-%d%d%-%d%d %d%d:%d%d:%d%d") .. "$"
 
+    local orphan_highlight = {}
```

Comment:
Should be plural, as it's a list of orphans, not a single orphan highlight. `orphan_highlights`